packages/aws-client/src/services.ts

```diff
@@ -15,7 +15,7 @@ import { SNS } from '@aws-sdk/client-sns';
 import { STS } from '@aws-sdk/client-sts';
 import { ECS } from '@aws-sdk/client-ecs';
 import { EC2 } from '@aws-sdk/client-ec2';
-import * as AWS from 'aws-sdk';
+import { ElasticsearchService } from '@aws-sdk/client-elasticsearch-service';
```

Comment:
Discussed in huddles the difference between services and clients. From their docs https://github.com/aws/aws-sdk-js-v3, this appears correct. The service should allow v2 and v3 interfaces. The naming here is a bit different than other services, which is interesting but appears correct https://docs.aws.amazon.com/AWSJavaScriptSDK/v3/latest/client/elasticsearch-service/